internal/backends/winit/accesskit.rs

```diff
@@ -459,22 +459,23 @@ impl AccessKitAdapter {
             builder.add_action(Action::Focus);
         }
 
-        let min = item
+        if let Some(Ok(min)) = item
```

Comment:
I think you can simplify this to `if let Some(min) = item.prop(…).and_then(|min| min.parse().ok())`